src/gui/sharemanager.cpp

```diff
@@ -283,6 +294,12 @@ void LinkShare::slotNameSet(const QJsonDocument &, const QVariant &value)
     emit nameSet();
 }
 
+void LinkShare::slotLabelSet(const QJsonDocument &, const QVariant &label)
+{
+    _label = label.toString();
```

Comment:
Should this one not be guarded and only emit a signal if `_label != label.toString()` ?